app/lib/url.rb

```diff
@@ -72,19 +70,4 @@ def self.local_image(image_name, host: nil)
   def self.organization(organization)
     url(organization.slug)
   end
-
-  # Ensures we don't consider serviceworker.js as referer
-  #
-  # @param referer [String] the unsanitized referer
-  # @example A safe referer
-  #  sanitized_referer("/some/path") #=> "/some/path"
-  # @example serviceworker.js as referer
-  #  sanitized_referer("serviceworker.js") #=> nil
-  # @example An empty string
-  #  sanitized_referer("") #=> nil
-  def self.sanitized_referer(referer)
-    return if referer.blank? || URI(referer).path == SERVICE_WORKER
```

Comment:
It says this is for the service worker, but do we still want to keep the check for `referrer.blank?` ?

Reply:
Thanks @citizen428. I've made the changes.